pkg/cmd/cli/cmd/deploy.go

```diff
@@ -317,7 +322,18 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
 		return fmt.Errorf("couldn't cancel deployment %s", strings.Join(failedCancellations, ", "))
 	}
 	if !anyCancelled {
-		fmt.Fprintln(out, "no active deployments to cancel")
+		if latest == nil {
+			// TODO: this could mean that a new deployment is forthcoming but hasn't
+			// yet been created; might not be worth trying to express in this
+			// context.
+			fmt.Fprintf(out, "no deployments are in progress\n")
+		} else {
+			timeAt := strings.ToLower(units.HumanDuration(time.Now().Sub(latest.CreationTimestamp.Time)))
```

Comment:
Spoke with @kargakis and we decided that there needs to be a separate PR to extract various things here into `pkg/util`.